nightmare_firmware/servos.cpp

```diff
@@ -157,7 +157,7 @@ int servo::read(){
 		if(val > 32767){             //fix overflow if 0 is exceded (angle < -120)
 			val -= 65536;
 		}
-		angle = fmap(val,0,1000,-120,120); //hard set angle
+		angle = map(val,0,1000,-120,120); //hard set angle
```

Comment:
Since all arguments are integer types isn't it going to use the arduino map function instead? Is this intended?

Reply:
yes because it is going to downscale a 0..1000 range to a 0..240 one, I left fmap there because it may be useful in the future